.github/workflows/repo-creation.yml

```diff
@@ -115,6 +125,24 @@ jobs:
         run: |
           REPO_URL=$(gh repo create $REPO_OWNER/$REPO_NAME --$VISIBILITY --clone)
           echo "repo_url=$REPO_URL" >> "$GITHUB_OUTPUT"
+      - name: Get project admins
+        run: |
+          
+      - name: Add admins to repo
+        id: add-admins
+        if: env.ADMINS
+        env:
+          GH_TOKEN: ${{ steps.get-workflow-token.outputs.token }}
+        run: |
+          for admin in $(echo $ADMINS | tr ", " "\n")
```

Comment:
what if they enter the list like `user1,user2,user3` or `user1,  user2,  user3`?

this is why i would not use text fields for this but yaml only